Dockerfile

```diff
@@ -3,42 +3,35 @@
 # production: runs the actual app
 
 # Build builder image
-FROM ruby:3.2.2-alpine as builder
-
-# RUN apk -U upgrade && \
-#     apk add --update --no-cache gcc git libc6-compat libc-dev make nodejs \
-#     postgresql13-dev yarn
+FROM ruby:3.2.2-bookworm as builder
 
 WORKDIR /app
 
-RUN addgroup -g 1000 -S appgroup \
-  && adduser -u 1000 -S appuser -G appgroup
+RUN addgroup --gid 1000 --system appgroup
+RUN adduser --uid 1000 --system appuser --gid 1000
 
-# Add the timezone (builder image) as it's not configured by default in Alpine
-RUN apk add --update --no-cache tzdata && \
-    cp /usr/share/zoneinfo/Europe/London /etc/localtime && \
-    echo "Europe/London" > /etc/timezone
+# Yarn doesn't have a native Debian package, so we need to download it from its own repo
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | gpg --dearmor -o /usr/share/keyrings/yarn-archive-keyring.gpg
+RUN echo "deb [signed-by=/usr/share/keyrings/yarn-archive-keyring.gpg] https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
 
-# build-base: dependencies for bundle
 # yarn: node package manager
-# postgresql-dev: postgres driver and libraries
 # git: to allow us to create the VERSION file
-RUN apk add --no-cache build-base yarn postgresql13-dev git
+# npm - so that we can run puppeteer via npx
```

Comment:
As a 'love letter' to our future selves 🥰 , can we would also add?:

```
# apt update: command to keep all of our packages up to date in Debian
# yarn: node package manager
# git: to allow us to create the VERSION file
# npm:  so that we can run puppeteer via npx
```